tests/ctst/package.json

```diff
@@ -18,7 +18,8 @@
     "prometheus-query": "^3.4.0",
     "proper-lockfile": "^4.1.2",
     "qs": "^6.13.0",
-    "scubaclient": "git+https://github.com/scality/scubaclient#^1.1.2"
+    "scubaclient": "git+https://github.com/scality/scubaclient#^1.1.2",
+    "uuid": "^11.1.0"
```

Comment:
From what i can see we add this dependency just to create a unique pod name, is that really needed? With ctst we have some utility function, used for instance to generate object names, that could be used instead?